docs/rst/user_manual/configuration.rst

```diff
@@ -781,9 +671,12 @@ It will use such profile for configuring the Participant.
 
     profile: participant_custom_configuration
 
+QoS
+---
 
-.. _user_manual_configuration_forwarding_routes:
+Participants support a ``qos`` **optional** tag to manually configure their :ref:`Topic QoS <user_manual_configuration_topic_qos>`.
```

Comment:
```suggestion
``participants`` support a ``qos`` **optional** tag to manually configure their :ref:`Topic QoS <user_manual_configuration_topic_qos>`.
```

Reply:
I disagree. `participants` makes it sound as if the tag was meant to be under the `participants` tag, when in reality it's meant to be under each participant.